src/components/blocked-patron/blocked-modal/BlockedModal.tsx

```diff
@@ -17,6 +17,10 @@ const BlockedModal: FC<BlockedModalProps> = ({ blockedStatus }) => {
 
   const { blockedModal } = getModalIds();
 
+  if (!blockedStatus || blockedStatus === "") {
+    return null;
+  }
```

Comment:
This is coming out of the blue :) 
Could you please write a comment why it is necessary?